ldap/servers/slapd/log.c

```diff
@@ -202,6 +202,7 @@ compress_log_file(char *log_name, int32_t mode)
     if ((source = fopen(log_name, "r")) == NULL) {
         /* Failed to open log file */
         gzclose(outfile);
+        close(fd);
```

Comment:
fd is closed by gzclose so you should  not close(fd) despite covscan warning ...